paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -50,7 +50,7 @@ abstract class PrepareResourcesTask : DefaultTask() {
   abstract val mergeAssetsOutput: DirectoryProperty
 
   @get:InputDirectory
-  @get:PathSensitive(PathSensitivity.RELATIVE)
+  @get:PathSensitive(PathSensitivity.ABSOLUTE)
```

Comment:
That is a bad change, this would make all remote build cache hits have a miss. If anything, we should be using `PathSensitivity.NONE` since we do not really care where the directory is, other than the contents of it.

Reply:
@liutikas sorry for delay, I had a doubt and I tried again: using `PathSensitivity.NONE` would fails on the same error. The task `preparePaparazziReleaseResources` is still resolved from the cache from the steps described on #495: 

<img width="936" alt="image" src="https://user-images.githubusercontent.com/773377/183098204-68058929-7577-4120-983d-e15245768e79.png">

Reply:
I think the reason of this task not working well with Gradle cache is because the actual jar file is downloaded and stored via the Gradle dependency manager, it's not an output of the task. The values stored into the Gradle cache for this task is the path to the jar file. Using `ABSOLUTE` will cause some cache-miss but will fix this issue.

Reply:
@jrodbx and I discussed how to handle `platformDataRoot` properly. I'll create a PR on Monday with a fix that should address this issue without use of ABSOLUTE normalizer.